backend/Utils/EmailUtils.ts

```diff
@@ -19,7 +19,7 @@ ${message}
   return encode(str, true);
 }
 
-export async function sendEmail({ to, subject }) {
+export async function sendResetPasswordEmail({ to, subject }) {
```

Comment:
EmailUtils should not contain any logic specific to resetting passwords - it's a file used by any backend code that needs to send an email for any reason. Please revert the renaming of this function

Reply:
This makes sense.